infra/command/gen-coverage-report

```diff
@@ -52,16 +73,14 @@ do
 done
 
 # Capture initial zero coverage data
-# --ignore-errors mismatch: lcov bug in gcc 13, 14
 "${LCOV_PATH}" -c -i -d "${BUILD_WORKSPACE_PATH}" --gcov-tool ${GCOV_PATH} \
   -o "${RAW_BASE_COVERAGE_INFO_PATH}" \
-  --ignore-errors mismatch
+  ${IGNORE_ERRORS_MISMATCH_FLAG}
```

Comment:
`--ignore-errors mismatch` changed to `--ignore-errors mismatch --ignore-errors negative`.
is this OK ?

Reply:
I tested it, but it does not change result. So it's OK.